overrides/assets/stylesheets/home.css

```diff
@@ -516,6 +516,61 @@ footer nav.md-footer__inner {
   padding: 0;
 }
 
-footer .md-footer-meta .md-footer-meta__inner {
+.md-footer-meta{
   padding: 1em 0;
+}
+.md-footer-copyright{
+  margin-right: 18px;
+}
+@media (max-width: 1024px){
+.md-footer-copyright{
+  margin: auto;
+  text-align: center;
+}
+.md-social{
+  margin: auto;
+}
+}
+@media (max-width: 861px) {

```

Comment:
Between this PR and the other responsiveness one it seems like we have used three screen width queries with different widths so far...

Would it be possible to standardize to maybe 2 widths to check? One for medium and one for small screen sizes?